src/libraries/System.Collections/src/System/Collections/Generic/HashSet.cs

```diff
@@ -1303,12 +1325,15 @@ private void SetCapacity(int newSize)
             }
 
             int[] newBuckets = new int[newSize];
+#if BIT64
+            _fastModMultiplier = HashHelpers.GetFastModMultiplier((uint)newSize);
```

Comment:
Won't this introduce issues while the hash table is being resized?

Reply:
Wouldn't a local multiplier and future multiplier be the same value?

Reply:
(Looking again, this seems fine because nothing there should throw and leave _fastModMultiplier in a bad state.)